tools/typespec-bump-deps/package.json

```diff
@@ -1,16 +1,21 @@
 {
   "name": "@azure-tools/typespec-bump-deps",
-  "version": "0.9.7",
+  "version": "0.9.8",
   "description": "Tool to grab latest @next version for all Typespec packages",
   "main": "dist/index.js",
   "type": "module",
   "bin": {
     "typespec-bump-deps": "./cmd/cli.mjs"
   },
+  "files": [
+    "dist",
+    "cmd/cli.mjs"
+  ],
   "scripts": {
     "watch": "tsc -p ./tsconfig.build.json --watch",
     "build": "tsc -p ./tsconfig.build.json",
-    "clean": "rimraf dist/ temp/",
+    "clean": "rimraf dist/",
+    "prepack": "npm run clean && npm run build",
```

Comment:
The cause is this.

I use the same build ci from tsp-client, but I didn't aware that they've included the `build` step in `pack` but here I didn't.